install/Knative-custom-install.md

```diff
@@ -0,0 +1,222 @@
+
+  * The version of your
+    [`kubectl` CLI tool](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+    must be v1.10 or newer.
+
+## Installing Istio
+
+Knative depends on [Istio](https://istio.io/docs/concepts/what-is-istio/) for
+the service mesh. You can choose the default package to install the Istio service mesh with
+[automatic sidecar injection](https://istio.io/docs/setup/kubernetes/sidecar-injection).
+You should install the default package if you are just getting started with Knative.
+
+Alternatively, you can choose to install Istio with manual sidecar injection or with support for
+[Custom Resource Definitions (CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/).
+
+### Choosing an Istio installation package
+
+You can choose from the following installation packages to install the Istio
+service mesh on you cluster:
+
+| Istio Package Filename | Description       |
+| ---------------------- | ----------------- |
+| [`istio.yaml`][a]             | Default: Configured for Automatic Sidecar Injection. |
+| [`istio-crds.yaml`][b]        | Support for Custom Resource Definitions. **Q: This have to do with "Pilot"? Why choose this?** |
+| [`istio-lean.yaml`][c]        | Configured for Manual Sidecar Injection. **Q: When or why should users choose this?** |
+
+[a]: https://github.com/knative/serving/releases/download/v0.2.2/istio.yaml
+[b]: https://github.com/knative/serving/releases/download/v0.2.2/istio-crds.yaml
+[c]: https://github.com/knative/serving/releases/download/v0.2.2/istio-lean.yaml
+
+### Installing Istio installation packages
+
+1. Install an Istio package with the `kubectl apply` command by specifying the
+   package filename:
+    ```bash
+    kubectl apply --filename https://github.com/knative/serving/releases/download/v0.2.2/[FILENAME].yaml
+    ```
+
+    where `[FILENAME]` is the name of the Istio package that you want to install. Example:
+    `https://github.com/knative/serving/releases/download/v0.2.2/istio.yaml
+    `
+1. Label the default namespace with `istio-injection=enabled`: **Q: For which install package is this needed? Or is this needed for all Istio installs?**
+    ```bash
+    kubectl label namespace default istio-injection=enabled
+    ```
+1. View the status of your Istio installation. It might take a few
+   seconds so rerun the following command until all of the pods show a
+   `STATUS` of `Running` or `Completed`:
+    ```bash
+    kubectl get pods --namespace istio-system
+    ```
+
+    > Tip: You can append the `--watch` flag to the `kubectl get` commands to
+      view the pod status in realtime. You use `CTRL + C` to exit watch mode.
+
+## Installing Knative components
+
+You can install each Knative component independently or use the default packages
+to install multiple components. The individual component packages exclude
+an observability plugin but you have the option to install one later.
+
+### Choosing Knative installation packages
```

Comment:
Resolving this comment and will add @mdemirhan to the PR I open for the in-progress edit Im working on (#447).  [A bit delayed since I'm working my way back through all the changes I made initially for 0.1 and am now rewriting it all again to align with the new behavior in 0.2.]